mne/channels/tests/test_channels.py

```diff
@@ -257,6 +270,77 @@ def test_read_ch_adjacency(tmp_path):
     savemat(mat_fname, mat, oned_as='row')
     pytest.raises(ValueError, read_ch_adjacency, mat_fname)
 
+    # Try reading all built-in FieldTrip neighbors
+    for name in get_builtin_ch_adjacencies():
+        ch_adjacency, ch_names = read_ch_adjacency(name)
+        assert_equal(ch_adjacency.shape[0], len(ch_names))
+
+
+def _download_ft_neighbors(target_dir):
+    """Download the known neighbors from FieldTrip."""
+    import pooch
+    import requests
+    pooch.get_logger().setLevel('ERROR')  # reduce verbosity
+
+    # The entire FT repository is larger than a GB, so we'll just download
+    # the few files we need.
+    def _download_one_ft_neighbor(neighbor):
+        fname = neighbor.fname
+        try:
+            pooch.retrieve(
+                url=f'https://github.com/fieldtrip/fieldtrip/raw/master/'
+                    f'template/neighbours/{fname}',
+                known_hash=None,
+                fname=fname,
+                path=target_dir,
+            )
+        except requests.HTTPError as e:
+            if e.response.status_code == 404:
+                print(f'{fname} only ships with MNE-Python, but not '
+                      f'with FieldTrip')
+            else:
+                raise
```

Comment:
seems like this is not stringent enough; I think it would miss, for example, if fieldtrip simply renamed an adjacency file. Do we have a constant somewhere that says what subset of `_BUILTIN_CHANNEL_ADJACENCIES` originate from fieldtrip (or the complement, which ones do not)?  If so, we could only try to download those that we expect to actually be there, and not bother with the 404 `except` clause.

Reply:
I had thought about adding this later on to keep the diff of this PR smaller but now that I read your comment, maybe we should have it right away. I will add this. Thanks for your suggestion!

Reply:
I have addressed this in c35e9c9